Testscripts/Windows/STORAGE-HotAdd.ps1

```diff
@@ -5,7 +5,7 @@ param([String] $TestParams,
       [object] $AllVmData,
       [object] $CurrentTestData)
 
-$SETUP_SCRIPT = ".\TestScripts\Windows\AddVhdxHardDisk.ps1"
+$SETUP_SCRIPT = ".\TestScripts\Windows\AddHardDisk.ps1"
```

Comment:
This is the totally different script. Please provide why you want to change?

Reply:
Yes, this is original script bug. We have STORAGE-HotAdd.ps1 and STORAGE-HotAddVhdx.ps1 two scripts, STORAGE-HotAdd.ps1 is used for adding vhd file, need to call ".\TestScripts\Windows\AddHardDisk.ps1" but not ".\TestScripts\Windows\AddVhdxHardDisk.ps1".

The script STORAGE-HotAddVhdx.ps1 calls ".\TestScripts\Windows\AddVhdxHardDisk.ps1".